openlibrary/plugins/worksearch/code.py

```diff
@@ -125,6 +126,8 @@
     'random': 'random_1 asc',
     'random asc': 'random_1 asc',
     'random desc': 'random_1 desc',
+    'random.hourly': lambda: f'random_{datetime.now():%Y%m%dT%H} asc',
+    'random.daily': lambda: f'random_{datetime.now():%Y%m%d} asc',
```

Comment:
https://stackoverflow.com/questions/53449308/why-dont-f-strings-play-nicely-with-dictionaries

Reply:
https://stackoverflow.com/questions/14419925/storing-lambdas-in-a-dictionary

Reply:
Shouldn't affect here; the lambda issue was a problem because they were trying to do an assignment inside the lambda. And the dictionaries issue seems like it's about having dictionary access inside the `f`-string; not about having an `f`-string inside a dictionary.

The main benefit of this approach is that now all the sort options (but 1 :P) are easily defined in this dict, and easy for someone to look for/read :)